sources/Inputs/include/Behaviours.h

```diff
@@ -178,9 +179,15 @@ struct StaticVarCompensator {
    * @param USetPointMax the high voltage set point of the SVarC
    * @param b0 the initial susceptance value  of the SVarC
    * @param slope the slope (kV/MVar) of the SVarC
+   * @param hasStandByAutomaton whether the SVarC has the StandByAutomaton extension
+   * @param hasVoltagePerReactivePowerControl whether the SVarC has voltage per reactive power control information
+   * @param regulatedBusId regulated Bus Id
+   * @param connectedBusId connected Bus Id
+   * @param UNomRemote the nominal voltage of regulated bus
```

Comment:
If remote or always ?

Reply:
as above. I'll be more specific in the comment